docker-compose.yml

```diff
@@ -9,11 +9,11 @@ services:
         published: 4200
     environment:
       PROTOCOL: 'http:'
-      HOSTNAME: 0.0.0.0
+      HOSTNAME: localhost
       PORT: 4200
       PATHNAME: /
       API_ROOT: https://app.stable.retail.backbasecloud.com/api
-      AUTH_URL: https://identity.dev.sdbx.live.backbaseservices.com/auth/realms/retail
+      AUTH_URL: https://identity.dev.sdbx.live.backbaseservices.com/auth
```

Comment:
Is it save remove the realms of this url?

Reply:
the rest of the url is added in the app. At the end you only provide api auth url.